src/test/java/com/amihaiemil/eoyaml/AllYamlLinesTest.java

```diff
@@ -28,23 +28,51 @@
 package com.amihaiemil.eoyaml;
 
 import com.amihaiemil.eoyaml.exceptions.YamlReadingException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.List;
+
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 
 /**
  * Unit tests for {@link AllYamlLines}.
  * @author Mihai Andronache (amihaiemil@gmail.com)
  * @version $Id$
  * @sinve 1.0.0
  */
+@RunWith(Parameterized.class)
```

Comment:
@zsandoz Please remove this. I don't like parameterized tests... if a test fails, it's never clear which parameter cauzed the failure.

Furthermore, most tests in this class are using ``guessIndentation = false``, I'm not sure it makes sense for them to also run with ``guessIndentation = true``.

Reply:
Done